File: js/alternating-sums.js
function alternatingSums(a){
    let sumEven = 0
    let sumOdd = 0

    for (let i=0 ; i<a.length ; i++){
        if (i%2 == 0){
            sumEven += a[i]
        }else{
            sumOdd += a[i]
        }
    }
    return [sumEven, sumOdd]
}

let a = [50, 60, 60, 45, 70]
console.log(alternatingSums(a))

File: go/alternating-sums/alternating-sums.go
package main

import(
	"fmt"
)

func alternatingSums(a []int)[]int{
	sumEven := 0
    sumOdd := 0

	for idx, val := range a{
		if idx%2 == 0{
            sumEven += val
        }else{
            sumOdd += val
        }
	}
	return []int{sumEven, sumOdd}
}

func main() {
	a := []int{50, 60, 60, 45, 70}
	fmt.Printf("The answer for the array %v is %d", a, alternatingSums(a))
}